### DISCUSSION TITLE
Not working on embedded videos

### DISCUSSION THREAD

**Discussion by Luusername** [aclap-dev/video-downloadhelper#390] [Help and Q&A]
### Your question

Used to work fine, now it's not working anymore on embedded videos on joyclub.de

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: de
license: unset
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:109.0) Gecko/20100101 Firefox/115.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac13",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/xxx",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.1.1-tessus"
}

**Comment by Mattmjs** [comment]
There is a video on the homepage that works, is that what you're referring to. It works on this one.

**Comment by Luusername** [comment]
Talking about the videos of the users AFTER your login.
![NSFW](https://github.com/aclap-dev/video-downloadhelper/assets/55045299/9b72cecb-b1aa-4e59-bb33-edbe17fa18ff)

**Reply by Mattmjs** [reply]
Is DRM in use like this image:

![DRM icon](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/8efaf847-09ff-41ee-8b29-208747de967c)

**Reply by Luusername** [reply]
no

**Reply by Mattmjs** [reply]
I usually ask for the hit details, but you've got no detection so you might have to record the network in the developer tools.
Go to the video and make sure the network is empty. Then reload the page, after that share the results as an attachment.

![3-10-23](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/4c441812-76d4-4ac2-abc8-050f4d28ddfb)
